### DISCUSSION TITLE
Latest Version

### DISCUSSION THREAD

**Discussion by GitCat3** [Mr-X-GTA/YimMenu#37] [Q&A] [answered]
Does this work with the latest version of gta online? I saw they recently broke it, and I was wondering if it is fixed and still undetectable?

**Comment by Mr-X-GTA** [comment]
Yes, it does work

**Reply by GitCat3** [reply]
> Yes, it does work

I am using the latest version with xenos injector, and when I hit online, the game crashes after a minute or so

**Reply by GitCat3** [reply]
here is my `cout.log` if that helps
[cout.log](https://github.com/user-attachments/files/17709285/cout.log)

**Reply by Mr-X-GTA** [reply, marked as answer]
Use [FSL](https://www.unknowncheats.me/forum/grand-theft-auto-v/616977-fsl-local-gtao-saves.html)

**Reply by GitCat3** [reply]
> Use [FSL](https://www.unknowncheats.me/forum/grand-theft-auto-v/616977-fsl-local-gtao-saves.html)

ah so it just doesn't work with regular rockstar saves anymore?

**Reply by GitCat3** [reply]
> Use [FSL](https://www.unknowncheats.me/forum/grand-theft-auto-v/616977-fsl-local-gtao-saves.html)

Sorry to bother you again, but when I use FSL there are a bunch of cars missing from the various online stores in gtao, is there a more updated version of FSL that has these?

**Reply by GitCat3** [reply]
Interestingly I got time menu and FSL running on my steam deck and the in game stores had all the normal items, nothing was missing

**Reply by Mr-X-GTA** [reply]
Do you have any Lua scripts running that could cause this?

**Reply by GitCat3** [reply]
I have extras adding and ultimate menu

**Reply by Mr-X-GTA** [reply]
Disable them and see if it fixes it

**Reply by GitCat3** [reply]
> Disable them and see if it fixes it

That was the answer, thanks

**Reply by mirataen** [reply]
FSL updated to 1.4 but not released yet!

**Comment by FLIMBO557** [comment]
the business function does not work and does not add supplies to all factories.
you can no longer enter the garage after restoring the vehicle and vehicle modifications are also not saved

**Comment by grizzlybear1290** [comment]
Hey how do I use this menu

**Reply by grizzlybear1290** [reply]
And is there anyway too have it a .xex

**Reply by grizzlybear1290** [reply]
https://discord.gg/gAbWnKjQ

**Reply by tupoy-ya** [reply]
> And is there anyway too have it a .xex

##### This reply is based on the assumption that `.xex` means xbox 360 support.

No, GTA Online legacy does not mean it will run on the xbox 360.
If you wanted to port it, you will have to back port it to the prehistoric compiler xbox 360 uses, and obviously change all the native hashes, patterns, classes, globals, locals, etc., also ImGui doesn't compile with the xbox 360 compiler, other dependencies probably don't either, so you will have port them as well.
